lib/queue.rb

```diff
@@ -1,31 +1,36 @@
+require_relative "linked_list"
+
 class Queue
```

Comment:
This works, but isn't implemented with a circular buffer.  So it's not what's asked for.